docs/install.rst

```diff
@@ -1,26 +1,126 @@
+.. _installing-astropy:
+
 ************
 Installation
 ************
 
-.. _installing-astropy:
+.. _installing-astropy-with-conda:
 
-Installing ``astropy``
-**********************
+Installing Python and Astropy with Conda
+****************************************
 
-If you are new to Python and/or do not have familiarity with `Python virtual
-environments <https://docs.python.org/3/tutorial/venv.html>`_, then we recommend
-starting by installing the `Anaconda Distribution
-<https://www.anaconda.com/download/>`_. This works on all platforms (linux,
-Mac, Windows) and installs a full-featured scientific Python in a user directory
-without requiring root permissions.
+There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
+The easiest way to manage a Python installation together with an Astropy installation is with the ``conda`` package manager.
+If you already have Python and ``conda`` working you can skip the next section.
+However, if you are using the Anaconda Python Distribution, we recommend you still install miniforge as described below.
 
-Using pip
-=========
+Installing miniforge
+====================
+
+We recommend installing Python with `miniforge <https://github.com/conda-forge/miniforge/#miniforge>`__.
+This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``.
+
+First, download the installer for your system and architecture from the links below:
```

Comment:
I am not sure if we want to maintain a list of installers here. We are not a miniforge documentation. I don't feel like this is our job.

Reply:
also, installers will change and their locations will move...

Reply:
miniforge not having nice doc is not our problem to solve, or if you want to solve that, it isn't here. I am okay with suggesting to use miniforge but I do not want our doc to turn into their doc.

Reply:
> and the miniforge documentation is rubbish to non-existent

@Cadair - Is this what you were talking about?
https://github.com/conda-forge/miniforge/blob/main/README.md

I agree your version is prettier and nicer, but it seems like that README has the basic information on download and install. Maybe it's good enough?

Reply:
I think it really sucks if we make users click out to an inferior set of documentation (clicking out alone is friction) vs keeping like 6 links in our docs. If we contribute upstream in the future and make things better then sure let's get rid of it.

Reply:
Also: https://github.com/conda-forge/miniforge/issues/76

Reply:
Since that issue to make a nice download page has been open for nearly 4 years, I think we should just use the nice one that @Cadair has create for astropy. Maybe if this is brought to their attention they will use and maintain it, but in the meantime let's not stall this useful PR!